docs/pages/tutorial/helloworldscript.md

```diff
@@ -41,22 +41,34 @@ data:
 
 Next, we will load the configuration file into the configuration repository:
 * Copy the configuration file to the `metadata/in/new` folder
-* Navigate to the `bin` folder
-* Load the configuration file via the `bin/iesi-metadata.sh` (or `iesi-metadata.cmd` on Windows) command [![info](/{{site.repository}}/images/icons/question-dot.png)](/{{site.repository}}/pages/operate/operate.html)
+* Navigate to the `bin` folder and open the terminal (or command prompt on Windows) in this folder
+* Load the configuration file via the `./iesi-metadata.sh` (or `iesi-metadata.cmd` on Windows) command [![info](/{{site.repository}}/images/icons/question-dot.png)](/{{site.repository}}/pages/operate/operate.html)
 
+Linux/Mac
 ```bash
-bin/iesi-metadata.sh -load -type general
+./iesi-metadata.sh -load -type general
 ```
+Windows
+```bash
+./iesi-metadata.cmd -load -type general
+```
+
+This command saves your new `HelloWorldScript` script in the database in the backend. You will now be able to access it with the next command(s).
```

Comment:
I would go with: This allows you to store/load your created 'HelloWorld'-script in the configured central repository. You will now be able to execute your script with the next command(s).